posthog/models/organization.py

```diff
@@ -312,6 +312,48 @@ def validate_update(
             if membership_being_updated.level > self.level:
                 raise exceptions.PermissionDenied("You can only edit others with level lower or equal to you.")
 
+    def get_scoped_api_keys(self):
+        """
+        Get API keys that are scoped to this organization or its teams.
+        Returns a dictionary with information about the keys.
+        """
+        from posthog.models.personal_api_key import PersonalAPIKey
+        from django.db.models import Q
+        from posthog.models.team.team import Team
+        from django.utils import timezone
```

Comment:
All the `django` imports should be at the top level of the file (the `posthog.models` ones maybe not, not sure if they cause a circular import here)